docs/api/methods.jmd

```diff
@@ -313,6 +313,20 @@ A `resizeInfo` object, which will be passed to the supplied function, contains t
 
 ]) }}
 
+{{ api_method("getParams", "getParams (params[, id])", "Get the parameters for an upload request. Pass in a file id to make the parameters specific to that file.",
+[
+    {
+        "name": "params",
+        "type": "Object",
+        "description": "The parameters to include in the upload request."
```

Comment:
@seltix5 this is incorrect as `getParams` method doesn't need this parameter. Can you please correct this?.
I'll be happy to write tests for this

Reply:
done ^^